src/metabase/models/card.clj

```diff
@@ -46,6 +47,13 @@
       first
       :dashboards))
 
+(defn url
+  "Return a URL that can be used to access this Card."
+  [{:keys [id]}]
+  (when id
+    (str (setting/get :-site-url) "/card/" id)))
```

Comment:
Lets pick a route here and clean this up one way or the other.  I don't want to have 2 places where these urls are being formed in the code.

My preference is to continue using the `utils.urls` ns because I don't like having presentation specific code in model classes, but either will work.